### ISSUE TITLE
confusion continues

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1972]
**confusion continues**

One of the prospective publisher in Russia is not allowed by their layers to register as a publisher refering to GBIF.org a need for MoU to be signed beforehand.

I will handle this now, but this indicates that two pages may benefit from clearer message.

Become a publisher needs clearer wording of the "if your country...." senstence. Somthing like "National participation is not a condition for becoming a publisher, insutution can register and seek endorsement at any point".

A mirror clarification with stronger and clearer words might be needed at the national participation page, which also says about internation organisations. Something like this there:

IMPORTANT This page describes the particpation process four countries and large international organizations and consortia. Please remember that museum, universities, collections, reserves and other data holding institutions can register and seek enrosement at any point here [LINK], and this process is independent and is no conditional on national participation.

I have to explain this simple fact alsmot every time I present, the question "is not particpation required?" is very common. This fact was a "happy discovery" in Russia, Ukraine, Belarus, Georgia, Lativia recently. I hope that higher clarity at the website would trslate in faster growth of publishers

-----

User provided contact info: @dschigel
System: Firefox 60.0.0 / Windows 10.0.0
User: [See in registry](https://www.gbif.org/api/feedback/user/1e8fb4716cacde5a24a62eeafa0c3005:2519f98d4c36b775278226caf87f8c54edc50a0fff554cfd55928b5ee82ded877ae8280a1d81fbc8327fc08cbaadfe2be298826c6f09ac7f7075f6af960ec955)
Referer: https://www.gbif.org/become-a-publisher
Window size: width 1922 - height 1073
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2019-05-23T09:16:48.075Z',mode:absolute,to:'2019-05-23T09:22:48.075Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2019-05-23T09:16:48.075Z',mode:absolute,to:'2019-05-23T09:22:48.075Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by dschigel** [comment]
@ahahn-gbif @timhirsch I am trying to fix the Russian confusion with our publisher agreement, which, in fact does not help me: the agreement text refers to national participation and MoU all the time https://www.gbif.org/terms/data-publisher. A prospective publisher, from a member coutnry or not,  must accept data publisher agrement during the application process. The agreement itself refers to MoU, which does not apply to the registered publishers from the non-member countries. Is not it somehow circular, leaveing non-member publihsers at the uncharted maps? I thought to use our Publisher agreement as a clarification document, but I will hold for now.

**Comment by timhirsch** [comment]
I agree there is some room for confusion but it is much easier to change the 'become a publisher' text than the publisher agreement which is a legal document. On the former I have similar concerns and feel we need to separate the form joining process through the MoU and the data publishing process, but at the same time we do not want to undermine the role of nodes in coordinating national institutions as we then weaken the reason for countries to join. I will look at this wording and try to suggest changes tomorrow. Regarding the publisher agreement, we may argue for removing some or all of the preamble so that it goes straight to the terms and conditions specific to data publishers - but we would need to take legal advice before doing that.
